styles.css

```diff
@@ -0,0 +1,133 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+#back-images-intro {
+  width: 100%;
+  height: 100%;
+  padding-top: 24px;
+  background-image: url('./images/header-shapes\ mobile.svg');
+  background-repeat: no-repeat;
+}
+
+.connectlink {
+  padding: 12px;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  color: #7f8cff;
+}
+
+h1 {
+  padding-left: 12px;
+  width: 327px;
+  height: 150px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding-top: 206px;
+  color: #172b4d;
+}
+
+header {
+  background: white;
+  box-sizing: border-box;
+  left: 0;
+  padding: 24px;
+  position: fixed;
+  top: 0;
+  z-index: 1000;
+  width: 100%;
+}
+
+html {
+  font-family: 'Poppins', serif;
+  font-size: 16px;
+}
+
+.intro {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.intro-text {
+  width: 327px;
+  height: 146px;
+  padding: 12px;
+  box-sizing: border-box;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  border-radius: nullpx;
+}
+
+.logonav {
+  background: #e5e5e5;
+  list-style-type: none;
+}
+
+main {
+  background-image: url("./images/header-background-mobile.svg");
+  background-repeat: no-repeat;
+}
+
+nav ul {
+  list-style-type: none;
+}
+
+nav ul li {
+  display: inline-block;
+  margin: 0 2.5vw;
+  list-style-type: none;
+}
+
+section {
+  box-sizing: border-box;
+  min-height: 100vh;
+  padding: 40px 20px;
+  width: 100%;
+}
+
+.icons li {
+  margin: 0 2.5vw;
+  list-style-type: none;
+}
+
+.icons {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  list-style-type: none;
+}
+
+.icons button {
+  background: none;
+  border: none;
+}
+
+#logonav {
+  display: flex;
+  justify-content: space-between;
+  /* stylelint-disable-next-line no-missing-end-of-source-newline */
+}
```

Comment:
Kindly enable this `stylelint` rule again. All you need to do is add a new, empty line at the very end of the file for this linter check to pass.